### PR TITLE
feat: index ibm cognitive security

### LINKED ISSUE #730: Index IBM Security
~~Repo: https://github.com/carbon-design-system/ibm-security~~

~~Storybook: https://ibm-security.carbondesignsystem.com~~

Repo: https://github.com/carbon-design-system/ibm-cloud-cognitive

Path: https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/security

Storybook: https://carbon-for-ibm-products.netlify.app/?path=/story/security

Name: IBM Security

External docs? https://pages.github.ibm.com/cdai-design/pal

The page layouts could be our first "templates" that we index?

We don't need to index assets tagged as "Legacy". No need to index library navData or asset docs yet as we want to prove that with a Carbon core-maintained library first.

- [ ] Platform squad to open initial indexing PR in remote repo using the one-carbon.yml-per-library indexing approach
- [ ] Identify and share gaps with David, Elicia, and Cameron
- [ ] Open draft PR in this repo to register the library
- [ ] Once gaps are filled (e.g. all assets indexed with demo links, tags, status, thumbnail, etc.) and once the remote PR is merged, merge the platform draft PR

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #730 

Repo PR: https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2057

## TESTING: 
- Filter by `IBM Security`: http://localhost:3000/catalogs/components?maintainer[]=ibm-security
- You should see 11 indexed components with thumbnails

Screen Shot 2022-07-07 at 9 48 45 AM

TODOS: 

- [x] @aubrey-oneal double check tags
- [x] @jeanservaas thumbnails
- [x] update ref before merging
- [x] commit cached files (once updated ref)

### CODE REVIEW

**Comment by alisonjoseph** [comment]
Looks like patterns and templates aren't displaying? Is this a bug with our platform, or something with indexing?

**Comment by andreancardona** [comment]
> Looks like patterns and templates aren't displaying? Is this a bug with our platform, or something with indexing?

Yeah I couldn't figure out why 🤔 going to ask Matt about that

**Review by francinelucca — COMMENTED** [review]
Looks good to me! verified components, patters and template 🎉 , added minor suggestions on repo PR

**Comment by alisonjoseph** [comment]
This one is looking good to me too now 👍

**Comment by alisonjoseph** [comment]
LGTM other than the one missing thumbnail you already mentioned 😄 
Screen Shot 2022-07-07 at 2 57 53 PM

**Review by francinelucca — DISMISSED** [review]
LGTM ✨  except for stack-notification not showing thumbnail (think we need to rename svg to `stacked-notification`). 
Also I don't think we should push these cached files from `andreancardona` org, can we remove them.
Also left some CRs on repo pr

**Review by alisonjoseph — DISMISSED** [review]
I can see the components in the list

Screen Shot 2022-07-13 at 1 12 47 PM

howver if I do a filter by Security I don't see any components?

Screen Shot 2022-07-13 at 1 13 34 PM

Might be because we updated sponsor to be maintainer?

File: services/web-app/data/libraries.mjs [hunk 1]
```diff
@@ -460,6 +460,14 @@ const devLibraries = {
     path: '/',
     group: carbonComponentsGroup
   },
+  'ibm-security': {
+    host: 'github.com',
+    org: 'andreancardona',
+    repo: 'ibm-cloud-cognitive',
+    path: '/packages/security',
+    sponsor: 'ibm-security',
```
Comment (alisonjoseph) [inline services/web-app/data/libraries.mjs:468]:
```suggestion
    maintainer: 'ibm-security',
```

**Review by alisonjoseph — APPROVED** [review]
LGTM! I see all components with their thumbnails now.

**Review by francinelucca — COMMENTED** [review]
LGTM!

**Review by francinelucca — APPROVED** [review]
✨

**Pull request merged**